### ISSUE TITLE
Convert ELF training data

### ISSUE THREAD

**Issue by alreadydone** [leela-zero/leela-zero#2220]
I briefly looked at the ELF training data from https://dl.fbaipublicfiles.com/elfopengo/v2_training_run/urls.csv

The data is separated into games; each record entry contains the sgf (list of moves), the visit counts ("policies", a list of 21x21 vectors, which are 19x19 padded by 0), and the outcome ("reward") from black's perspective. So to convert into LZ format, we just need to rebuild the positions from the sgf (this is already done in dump_supervised), alternate between black and white for the color planes, extract the middle 19x19 from the policy data and divide by the sum, and revert the outcome at white's terms.

I'm sure some people are more knowledgeable than me on what tool to use (json parser, etc. though it's simple enough to deal with the strings directly), but if no one takes it up, I'd give it a try.

I think Minigo also released their training data for one or more of their runs, but I haven't looked into them. (Are they human readable or need to be parsed by tensorflow like the weight files?)

**Comment by alreadydone** [comment]
Wait, I can't find the visit count of the pass move...

**Comment by l1t1** [comment]
what does the paper talk about it

**Comment by alreadydone** [comment]
I found the pass move; it's the very first entry in the (21x21)-dim vector. The ELF training data converter is now ready (see https://github.com/leela-zero/leela-zero/pull/2232) but you need to decompress the .xz file to use it, which is rather inconvenient and takes up 10x disk space. https://github.com/do-m-en/libarchive_cpp_wrapper seems very convenient to use; let me know if you have a better suggestion!

**Comment by l1t1** [comment]
maybe you can use pipeline with your converter
